js/views/userPageVw.js

```diff
@@ -344,8 +372,9 @@ module.exports = baseVw.extend({
         blocked = this.options.userModel.get('blocked_guids') || [],
         isBlocked = blocked.indexOf(this.pageID) !== -1;
 
-    //add blocked status to model
-    this.model.set('isBlocked', isBlocked);
+    this.model.set('isBlocked', isBlocked); //add blocked status to model
+    this.model.set('ctrlCmdKey', config.keyShortcutPrefix);
```

Comment:
Since config is available globally on Window, the template could just get it from there. No need to add it to the model and subsequently pass it into the template.